docs/content/guide/passport.md

```diff
@@ -1,163 +1,58 @@
-# Passport
+## Passport
 
-Fundamentally, this module is used for authenticating and authorizing users (a.k.a. Indentities) on Spica Client and API.
+Fundamentally, this module is used for authentication and authorization on Spica Client and API.
 
-## Identities
+### Authentication
 
-Identities are the main entities of your Spica. They are basically profiles with custom properties and login information. You can attach Policies (see below) and restrict their access to the Spica.
+Out of the box, Spica supports two different authentication stretagies. This document will explain the fundemantals of those stretegies.
 
-### Authentication With Spica
+#### Identity
 
-#### With API Call
+Fundemantally, Identities are the users of a Spica instance. It contains **identified** and **password** informations for login.
 
-Request URL:
+> Right after the installation, Spica creates a default identitiy to get you started. We suggest you to change at least its password before going live.
 
-> GET "{baseUrl}/passport/identify?password={password}&identifier={unique identifiier}"
+To create an Identitiy, navigate to **System** -> **Identities** in the left-hand menu.
 
-If the credentials on request URL are correct, Spica returns the following response:
+- Click "+" icon on the top right toolbar.
+- Fill the **Identifier** and **Password** field.
+- Press **Save**
 
-```json
-{
-  "issuer": "spica:passport",
-  "token": "JWT {jwt token}",
-  "schema": "JWT"
-}
-```
+##### Adding Additional Properties
 
-"`token`", created by your Spica instance, contains the following informations when decrypted on its payload:
+If you want to store additional informations on **Identities**, you can create custom fields. To create a custom field on Identities, navigate to **System** -> **Settings** in the left-hand menu.
 
-```json
-{
-  "_id": "5d14f92e0617f15b1bc7d1b2", // Id string of the Identity
-  "identifier": "spica",
-  "policies": [...], // Array of the identitiy's Policy Ids
-  "iat": 1562668335, // Creationg date of the token
-  "exp": 1562841135, // Expiration date of the token
-  "aud": "spica.io",
-  "iss": "https://example.com" // Issuer domain of this JWT token. Mostly this will be the base url of your Spica instance
-}
-```
+#### API Key
 
-To authonticate every request to Spica API, attach the `token` to the `Authorization` header of your request. If you send request with empty `Authorization` header, Spica returns `401 Unauthorized` error.
+Instead of Identity, API key is allows the machine to machine communication. The token, it provides doesn't have an expiration date so it can be used as long as it's intentionally deleted from Spica.
```

Comment:
```suggestion
Instead of Identity, the API key allows the machine to machine communication. The token, it provides doesn't have an expiration date so it can be used as long as it is not intentionally deleted from Spica.
```